### DISCUSSION TITLE
Sonicwall Device Type

### DISCUSSION THREAD

**Discussion by tecguru** [ktbyers/netmiko#2721] [Q&A]
I have a python script that I am using to ssh into a Sonicwall and run SSH commands to monitor the status of my BGP connections. It is currently working, but I feel that I could simply the script by using netmiko for the SSH part. I am not sure if Sonicwalls are support by netmiko and if so which device_type I would use. 

My Sonicwall is a NSA 4600

Thank you for your assistance. 
Denver

**Comment by ganeshyadav123** [comment]
Sonicwall is not supported by the Netmiko but we can use the Paramiko to ssh using the python script and scrape whatever we want

Check my repo below which ssh into each individual sonicwall and schedule the restarts

https://github.com/ganeshyadav123/Sonicwall-Scheduled-Rebooter

**Comment by ktbyers** [comment]
We should just add a SonicWall driver to Netmiko. So if someone wants to contribute it, that would be great.
